### PR TITLE
chore: tweak the strict typings fixes

### PR DESCRIPTION
**Pull request by petebacondarwin**

### CODE REVIEW

**Comment by JacobMGEvans** [comment]
Not sure why the "Create Pull Request Prerelease / Build & Publish a Prerelease to the Adhoc Registry" is failing I tried a rerun but no joy, I don't think it blocks this PR since it is a CI/CD (infra) issue with our GitHub workflow not the code changes.

**Review by JacobMGEvans — APPROVED** [review]
Looks great! 🎊

**Comment by petebacondarwin** [comment]
The CI run failure is because the branch is on my own fork, and it doesn't have the correct permissions to add a comment to the PR.

File: packages/wrangler/src/inspect.ts [hunk 1]
```diff
@@ -590,9 +590,18 @@ function logConsoleMessage(evt: Protocol.Runtime.ConsoleAPICalledEvent): void {
...
+        console.dir(args);
+        break;
+      case "table":
+        console.table(args);
+        break;
+      default:
+        // eslint-disable-next-line prefer-spread
+        console[method].apply(console, args);
+        break;
+    }
```
Comment (petebacondarwin) [inline packages/wrangler/src/inspect.ts:604]: Hmm, actually this is not quite right.
We need to have special conversion for the `dir` and `table` second parameters...

**Review by threepointone — APPROVED** [review]

**Comment by threepointone** [comment]
Thank you!

**Pull request merged**